documentosNecesarios.md

```diff
@@ -0,0 +1,28 @@
+# Documentos necesarios para publicar una página
```

Comment:
I feel like this is part of a larger guide that is about getting your page ready for publication. To me, the favicons are also _files_ that are necessary for publishing your page. I opened an issue (#10) to address this.